NativeUI.csproj

```diff
@@ -70,7 +70,7 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>copy $(TargetPath) %25userprofile%25\Dropbox\proyekt\cfx-server\resources\[gamemodes]\test\nativeui.net.dll /Y</PostBuildEvent>
+    <PostBuildEvent>copy /y "$(TargetPath)" "D:\Servers\GTA V - Script Testing\resources\GPSMenu"</PostBuildEvent>
```

Comment:
Can you actually remove this post-build event entry? It was left over by accident, it'd be sensible to get rid of it.